apps/frontend/src/hooks/useWalletConnect.ts

```diff
@@ -0,0 +1,27 @@
+import { useCallback, useEffect, useState } from 'react';
+
+import { WalletState } from '@sovryn/onboard-core';
+
+import { onboard } from '../lib/connector';
+
+export const useWalletConnect = () => {
+  const [wallets, setWallets] = useState<WalletState[]>([]);
+
+  const connectWallet = useCallback(() => {
+    onboard.connectWallet();
+  }, []);
+  const disconnectWallet = useCallback(async () => {
+    await onboard.disconnectWallet(wallets[0].label);
+  }, [wallets]);
+
+  useEffect(() => {
+    const sub = onboard.state.select('wallets').subscribe(setWallets);
```

Comment:
Feed subscription with already connected wallets when next opening page

```suggestion
    const sub = onboard.state.select('wallets').pipe(startWith(onboard.state.get().wallets)).subscribe(setWallets);
```

`import { startWith } from 'rxjs/operators';`